ifrs17-template/Test/ReimportTest.ipynb

```diff
@@ -0,0 +1,291 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "VOS67A7-B0Omk1s4TFC3LQ",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "<p style=\"font-weight:bold;\"> <span style=\"font-size: 36px\"> Reimport of Transactional Data Test</span> </p>"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#!eval-notebook \"../Initialization/InitSystemorphBaseToMemory\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Check Method"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public async Task<ActivityLog> ReimportAndCompare((string transactionalData, string importFormat) importData, DataNodeState[] dataNodeStates)",
+                "\n{",
+                "\n    var workspace = Workspace.CreateNew();",
+                "\n    workspace.Initialize(x => x.FromSource(DataSource).DisableInitialization<DataNodeState>());",
```

Comment:
also disable init for RawVariable and IfrsVariable

Reply:
No need to disable since I use the InitSystemorph**Base**ToMemory, but I will disable to be double sure.